programmers/난이도별/level01.크레인_인형뽑기_게임/dkdlelk99.py

```diff
@@ -0,0 +1,26 @@
+def solution(board, moves):
+    answer = 0
+    basket = []
+    new_board = []
+    for i in range(len(board)):
+        row = []
+        for j in range(len(board)):
+            if board[j][i] != 0:
+                row.append(board[j][i])
+        new_board.append(row)
```

Comment:
```suggestion
    boardRange = range(len(board))
    for i in boardRange:
        row = []
        for j in boardRange:
            if board[j][i] != 0:
                row += [board[j][i]]
        new_board += [row]
```
len이나 range 같은건 재활용할 수 있으면 재활용해주세요!